Guide-to-migrate-e2e-tests.md

```diff
@@ -0,0 +1,65 @@
+## Overview
+
+Currently, Oppia E2E tests are in a hybrid state where we have both WebdriverIO and Protractor test suites. The project **Migrate away from Protractor** aims to migrate all the existing E2E tests to WebdriverIO.
+
+This guide is to help contributors write/modify E2E tests in webdriverIO during the hybrid state.
```

Comment:
```suggestion
This guide is to help contributors write and modify E2E tests in webdriverIO during the hybrid state.
```

It's often unclear whether "/" means "and" or "or"

Reply:
Done.